Publications/Surveillance Capitalism.md

```diff
@@ -0,0 +1,107 @@
+# Surveillance Capitalism
+There are Thousands of companies spying on you with a similar level of sophistication as state Intelligence Agencies(CIA, NSA, GRU, etc.). This is mostly done to gather data for advertising purposes, but there are very serious implications to this data-gathering. 
+
+The methods of data-collection are numerous, and include hundreds of data points in order to uniquely identify you based on:
+
+- Location: Country / Location / Timezone (GPS and Known Location)
+- How you type and the language you use(Keystroke Dynamics) 
+- Browsing Habbits and History
+- IP Address / ISP
+- Web-Browser type, version and extentions(Browser Fingerprinting)
+- Operating System type and version 
+- What devices you use(Cross-Device Tracking)
+
+With this information, not only are these companies able to know more about you than you know yourself, but they can track you Online and Offline. They can tell who you live with, who your friends, associates, and coworkers are, what stores you go to, what events and places you go to, what beer you like, your politics, your sexuality, and other details that are incredibly intimate. With that information, they can control what you see, what they show you, or, as shown by the Cambridge Analytica scandal, they can manipulate(or attempt to) you and people like you with shocking precision. These profiles are mostly(99%) accurate once they have been created, and they can be extremely difficult to get away from, even if you use a VPN.
+
+This is concerning because when products are created insecure by design, it allows attackers, whether state-actors or arganized crime, to leverage those existing insecurities.
+
+## What Can We Do?
+On a group level, we can fight for improved Privacy and Data security laws. We can fight to fix a system intentionally broken [(Bruce Schneier)](https://ideas.ted.com/why-an-insecure-internet-is-actually-in-tech-companies-best-interests/). We can fight for GDPR-like lasws in the United states. There are many organizations dedicated to this fight, such as the Electronic Freedom Foundation(EFF).
+
+On an individual level, The main thing we can do is to minimize the data we are sending to these companies harvesting our data. One way we can do this is by using alternative software to popular products, such as "Free and Open Source Software"(FOSS). FOSS doesn't have the financial incentive to data-mine or violate your privacy, and the source-code is available to to public to audit. 
+
+So, here is a list of software(and some Hardware) that is highly recommended to help migration away from data-mining companies. 
+
+# Recommended Software
```

Comment:
Maybe we want to move everything below this line to it's own resource in the future. I'll go ahead and merge for now.